src/test/java/nextstep/payments/domain/PaidPaymentPolicyTest.java

```diff
@@ -0,0 +1,103 @@
+package nextstep.payments.domain;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+class PaidPaymentPolicyTest {
+
+    @Test
+    @DisplayName("유효한 수강료와 최대 수강 인원으로 PaidPaymentPolicy를 생성할 수 있다")
+    void createPaidPaymentPolicy() {
+        // given
+        long fee = 10000L;
+        int enrollmentLimit = 30;
+
+        // when
+        PaidPaymentPolicy policy = new PaidPaymentPolicy(fee, enrollmentLimit);
```

Comment:
```suggestion
        PaidPaymentPolicy policy = new PaidPaymentPolicy(10000L, 30);
```
굳이 로컬 변수로 구현하기 보다 위와 같이 하드코딩하는 것은 어떨까?